### PR TITLE
Whitespace Removed Line Counts [pr]

### PR DESCRIPTION
**Pull request by pdb493**

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?php
+
 namespace App;
 
 use Illuminate\Notifications\Notifiable;
-use Illuminate\Foundation\Auth\User as Authenticatable;
+//use Illuminate\Foundation\Auth\User as Authenticatable;
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:6]: test customer reply

**Comment by pdb493** [comment]
test customer summary

**Review by pbarsallo — APPROVED** [review]

File: app/User.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:6]: another customer reply

**Comment by pdb493** [comment]
> ;lakjsdf;lkjasdf;lkjasdf;lkjasdfl;kjasdf;lkajsdf
> 
> Image of Patrick R **Patrick R**
> 
> Was this helpful? [Yes](https://patrick-9a73.pullrequest.tech/reviews/ratings/801d6073-cb96-407a-8c40-d04bb7ad31c6/true) | [No](https://patrick-9a73.pullrequest.tech/reviews/ratings/801d6073-cb96-407a-8c40-d04bb7ad31c6/false)
> 
> > _Reviewers will be notified any time you reply to their comments or commit new changes._
> > _You can also request a full follow-up review from your [PullRequest dashboard](https://patrick-9a73.pullrequest.tech/dash/projects/1/pdb493/test/reviews/)._

asdfasdfasdf

File: app/User.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:6]: asdf

File: app/User.php [hunk 2]
```diff
@@ -23,13 +24,7 @@ class User extends Authenticatable
...
+        'password', 
+	'remember_tokenz',
     ];
 }
-
-
-
-
-
-//d
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:34]: this is a test customer reply submitted in a review (github)

File: app/User.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:6]: hey

File: app/User.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (pbarsallo) [reply app/User.php:34]: another test customer reply

**Pull request closed without merging**